code/__DEFINES/atmospherics.dm

```diff
@@ -136,15 +136,15 @@
 /// The natural temperature for a body
 #define BODYTEMP_NORMAL						310.15
 /// This is the divisor which handles how much of the temperature difference between the current body temperature and 310.15K (optimal temperature) humans auto-regenerate each tick. The higher the number, the slower the recovery. This is applied each tick, so long as the mob is alive.
-#define BODYTEMP_AUTORECOVERY_DIVISOR		11
+#define BODYTEMP_AUTORECOVERY_DIVISOR		14
 /// Minimum amount of kelvin moved toward 310K per tick. So long as abs(310.15 - bodytemp) is more than 50.
-#define BODYTEMP_AUTORECOVERY_MINIMUM		12
+#define BODYTEMP_AUTORECOVERY_MINIMUM		6
 ///Similar to the BODYTEMP_AUTORECOVERY_DIVISOR, but this is the divisor which is applied at the stage that follows autorecovery. This is the divisor which comes into play when the human's loc temperature is lower than their body temperature. Make it lower to lose bodytemp faster.
 #define BODYTEMP_COLD_DIVISOR				15
 /// Similar to the BODYTEMP_AUTORECOVERY_DIVISOR, but this is the divisor which is applied at the stage that follows autorecovery. This is the divisor which comes into play when the human's loc temperature is higher than their body temperature. Make it lower to gain bodytemp faster.
 #define BODYTEMP_HEAT_DIVISOR				15
 /// The maximum number of degrees that your body can cool in 1 tick, due to the environment, when in a cold area.
-#define BODYTEMP_COOLING_MAX				-100
+#define BODYTEMP_COOLING_MAX				-30
```

Comment:
So you can cool less in one tick, but you recover slower. Is that it?

Reply:
We reduced how fast you can become colder by lowering BODYTEMP_COOLING_MAX, no longer do we drop by -100 in a tick.
bodytemp_autorecovery_min was used as part of the older body temp calculation and in now only used on sub humans.
The BODYTEMP_AUTORECOVERY_DIVISOR was moved upward because the change steps would have been to large otherwise making temperature swing wildly. I found 12-18 to be good numbers with out taking a long time, or causing massive swings.